src/doc/src/reference/workspaces.md

```diff
@@ -137,11 +137,12 @@ members = ["path/to/member1", "path/to/member2", "path/to/member3/*"]
 default-members = ["path/to/member2", "path/to/member3/foo"]
 ```
 
-> Note: when a [root package](#root-package) is present,
-> you can only operate on it using `--package` and `--workspace` flags.
+> Note: If `default-members` is specified and a [root package](#root-package) is present,
+> you can only operate on the root package using `--package` and `--workspace` flags.
+> There is no way to specify the root package explicitly in the `default-members` field.
```

Comment:
Actually you can do that through `default-members = ["."]`.

Maybe we should remove this instead?